app.getclipboard.Clipboard.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>app.getclipboard.Clipboard</id>
+  
+  <name>Clipboard</name>
+  <summary>Cut, copy, and paste anything in your terminal</summary>
+  
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  
+  <description>
+    <p>
+      The Clipboard Project (CB for short) is an advanced clipboard manager that&apos;s made for your terminal. Cut, copy and paste anything, anytime, anywhere with unlimited capacity, clipboards, and history. You can also script CB to seamlessly work with your other favorite terminal tools. Plus, it seamlessly integrates with your existing system clipboards.
+    </p>
+    <p>
+      IMPORTANT! If you install the Clipboard Project with Flatpak, you'll likely be able to run it with the command "flatpak run app.getclipboard.Clipboard". However, this is long and hard to remember, and Flatpak doesn't yet offer a built-in solution so that you can just use "cb". So, to work around this issue, run the command "alias cb='flatpak run app.getclipboard.Clipboard'" so that the command "cb" does what you expect. Add that command to your terminal startup file like .bashrc so that it works every time you start the terminal.
+    </p>
+  </description>
+  
+  <icon type="remote" height="512" width="512">https://raw.githubusercontent.com/Slackadays/Clipboard/0.8.0/documentation/readme-assets/CB.png</icon>
```

Comment:
please avoid remote icons.

Reply:
how do you add a local icon? the only reason this uses a remote icon is because I could never get local ones to work because they're undocumented

Reply:
you install the icon in the proper xdg path and provide a .desktop file, installed in the right place. (with proper names)

https://github.com/flathub/flathub/wiki/App-Requirements#desktop-integration
(and subsequent links)

Reply:
the thing is, .desktop files don't apply here because this is a CLI app (`Do note that CLI applications do not require a .desktop file`)

Reply:
CLI apps aren't a use case for flatpak. also why does it access the display server then?

Reply:
cb requires x11/wayland access in order to access those clipboard systems, and by default it stores everything in `xdg-run/Clipboard`. the reason we need flatpak for this is because cb requires gcc 11+ which many systems don't have, so flatpak lets cb work on anything that supports flatpak, and users want to get cb on flathub too